Src/Witsml/Extensions/StringExtensions.cs

```diff
@@ -13,5 +14,25 @@ public static string NullIfEmpty(this string value)
         {
             return string.IsNullOrEmpty(value) ? null : value;
         }
+        
+        /// <summary>
+        /// Determines whether two specified string objects have the same value.
+        /// </summary>
+        /// The first string to compare.
+        /// The second string to compare.
+        /// <returns>True if the value of the <paramref name="firstValue" /> parameter is equal to the value of the <paramref name="secondValue" />, otherwise return false.</returns>
+        public static bool EqualsIgnoreCase(this string firstValue, string secondValue)
```

Comment:
Would it be better to make this an extensions of `Uri` instead of `string` to avoid calling `toString()` before every call of EqualsIgnoreCase?

Other than that this method seems to replicate `string.Equals(firstValue, secondValue, StringComparison.OrdinalIgnoreCase)` functionality.

Reply:
I fixed it again with == operator, because comparison is case sensitive  and includes more valid cases.

Reply:
The comparison is supposed to not be case sensitive, so the `==` operator cannot be used in this case.

Reply:
Sorry, I meant to say that == operator is case insensitive for Uri.

Reply:
The `==` operator is case sensitive for `Uri`.
![image](https://github.com/equinor/witsml-explorer/assets/66632214/c06e91ed-df57-4803-9722-27b7a6ded1e0)

Reply:
I tested these Uri "HTTPS://www.EXAMPLE.com/" or "HTTPS://WWW.example.com/" it is case-insensitive and everything else is case-sensitive. For example https://en.wikipedia.org/wiki/GIt is not found, but https://en.wikipedia.org/wiki/Git is correct.
If should it be case insensitive also after the slash in this case, so I will change it.

Reply:
Yes, it should be case insensitive after the slash. This was requested by Stian Steinsbø Lie due to disrepancies in how WITSML server URLs are entered across different systems interacting with WITSML servers. Thank you for clarifying.

Reply:
Corrected.